distributed/asyncio.py

```diff
@@ -127,10 +127,6 @@ def __init__(self, *args, **kwargs):
     async def __aexit__(self, type, value, traceback):
         await self.shutdown()
 
-    def __del__(self):
-        if self.status == 'running':
-            self.loop.asyncio_loop.run_until_complete(self.shutdown(fast=True))
```

Comment:
Hmm... This raises the question: can `__del__` even be called if some AioClient coroutines are still running?
In other words, perhaps we can put a `assert self.status != 'running'` in `__del__`?